tests/console/nvme_checks.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Smoke tests for NVM Express.
+#
+# - Install nvme-cli which is a tool to manage NVM Express
+# - Check sys path that should be hold details
+# - Check if we have any NVMe controllers on the PCI bus
+# - Check if we have NVMe devices
+# - Check if we have namespace
+# - Send Identify Admin Command to the NVMe controller
+# - Issue a Read command both using the helper and manually
+# - Compare it to what we have with a regular dd
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use scheduler 'get_test_suite_data';
+
+sub _check_basic_installation {
+    my $nvm_test_data = shift;
+    assert_script_run("lspci -nn | grep -i nvm");
+    assert_script_run("test -d $nvm_test_data->{nvme_sys_path} || nvme directory not found in $nvm_test_data->{nvme_sys_path}");
```

Comment:
Don't we miss `echo` before the message? Also, we could have used `fail_message` parameter of `assert_script_run` which will be better visible in openQA: http://open.qa/api/testapi/#_assert_script_run

Reply:
fixed